app/src/main/java/com/summer/practice/tvtracker/networking/CreateCallbackObject.kt

```diff
@@ -0,0 +1,30 @@
+package com.summer.practice.tvtracker.networking
+
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class CreateCallbackObject(
+    private val onSuccess: (movies: List<Movies>) -> Unit,
+    private val onError: (String) -> Unit
+){
+    fun createObject() = object: Callback<Results> {
```

Comment:
remove the class and leave only the createObject method, add the onSuccess and onError params to this method. Let's have a final version like this:
```
fun createCallback(
        onSuccess: (List<Movies>) ->Unit,
        onError: (String)->Unit
) {
   // object: Callback<Result>...
}
````